tests/Providers/Trustpilot/ServiceActionsTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace PodPoint\Reviews\Tests\Providers\Trustpilot;
+
+use PodPoint\Reviews\ActionsInterface;
+use PodPoint\Reviews\ApiClientInterface;
+use PodPoint\Reviews\Providers\Trustpilot\ServiceActions;
+use PodPoint\Reviews\Tests\TestCase;
+
+/**
+ * Class ServiceActionsTest
+ * @package PodPoint\Reviews\Tests\Providers\Trustpilot
+ */
+class ServiceActionsTest extends TestCase
+{
+
+    /**
+     * Constructor should be instance of APiClientInterface and it should assign to apiClient property.
+     */
+    public function testConstruct()
+    {
+        $action = new Mocked_ServiceActions($this->getMockedApiClient());
+
+        $this->assertInstanceOf(ActionsInterface::class, $action);
+        $this->assertInstanceOf(ApiClientInterface::class, $action->getApiClient());
+    }
+
+    /**
+     *  Making sure business id is setters and getters setting property as expected.
```

Comment:
You'll hate me for this one :joy: But there's an extra space before 'Making'

Reply:
I don't hate you mate, I hate the tools I am using. They should be able to detect this kind of issues.